hyppo/ksample/mean_embedding.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from numba import jit
+from scipy.stats import chi2
+from ._utils import _CheckInputs
+from .base import KSampleTestOutput, KSampleTest
+
+
+class MeanEmbeddingTest(KSampleTest):
+    r"""
+        Mean Embedding test statistic and p-value
+
+        The Mean Embedding test is a two-sample test that uses
+        differences in (analytic) mean embeddings of two data
+        distributions in a reproducing kernel Hilbert space in order
+        to determine how different the data are.
+
+        Notes
+        ____
+
+        The test statistic, like the Smooth CF statistic, takes on the following form:
+
+        :math: 'nW_n\Sigma_n^{-1}W_n'
+
+        As seen in the above formulation, this test-statistic takes the same form as
+        the Hotelling :math: 'T^2' statistic. However, the components are
```

Comment:
link to the Hotelling function using `:class:`

Reply:
added